shared/src/commonMain/kotlin/com.macaosoftware.sdui.app/marketplace/amadeus/auth/AuthViewModel.kt

```diff
@@ -19,6 +20,7 @@ class AuthViewModel(
     private val authPlugin: AuthPlugin
 ) : ComponentViewModel() {
 
+    val plugin = Plugin()
```

Comment:
We inject the dependencies using koin we don't create instances manually. An also that class Plugin should be removed, we don't need it.